README.md

```diff
@@ -76,6 +76,7 @@ Sample output is here
 |--order|-o|Sort order (default,time,file)|
 |--derived-data-path||Root path of DerivedData directory|
 |--truncate-at|-t|Truncate the method name with specified length|
+|--unique|Compact the duplicated results|
```

Comment:
the default behavior would be `--unique`.
So, non default options should be documented.

 `|--no-unique|Show duplicated location results|`